Robin/HW1_Robin/mock.css

```diff
@@ -0,0 +1,72 @@
+body{
+	background:url("asset/page-mock.png") no-repeat;
+	background-size: 1024px 885px;
```

Comment:
you can use shortcut property to conbine background and background-size together